_posts/2015-05-21-2015-so-far.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: "The year so far & 1.0 update"
+authors: [andyberry88]
+thumb: are-we-nearly-there-yet.png
+latest: true
+
+excerpt: "It's been a while since our blog saw some love. It's about time for an update a 1.0 release and what else the team have been up to."
+---
+
+It's been a while the blog has been some love and several months since we provided an update on progress towards a BladeRunnerJS 1.0 release, so here it is!
+
+## Man Down!
+
+<img class="pull-right" src="/blog/img/keep-calm-and-code.png" alt="Keep calm and code" width="200"/>
+
+At the start of the year [Phil Leggetter](https://github.com/leggetter), our Developer Evangelist, left the project to re-join [Pusher](https://pusher.com) as their Head of Developer Evangelism. Phil has done a fantastic job (don't get a big head Phil!) promoting the toolkit and guiding user experience decisions we've made while developing new features. He'll be missed and we wish him all the best in his new role.
+
+Phil's departure has made an impact in the capacity of the team, we're now 1 man down with the same amount of work. But it has encouraged us all to promote the toolkit and become 'devangelists' of sorts ourselves. So you'll hopefully see/hear more of the team at meetups, conferences and the writing for the BRJS blog.
+
+## 2015 so far
+
+We're getting ever closer to a final 1.0 release and the team have been working away finishing features and squashing bugs for 1.0. While this has been going on we've also attended various conferences to speak to developers about building complex webapps and the lessons we've learned from years of building them at Caplin.
+
+We were at QCon London at the start of March. This was a great event where we spoke to plenty of developers about their webapps and attended some great talks. 2 weeks later we were across the road from the QCon venue at DevWeek. Again this was a really great event with interesting talks and some great folk to talk to. Most recently we sponsored Jax Finance. While a small conference, compared to the previous, this too was very useful and we spoke to several very interesting developers who had encountered similar problems to Caplin with their own finance related webapps.
+
+There was also an internal project 'retrospective' where we invited internal developers at Caplin to the pub for some drinks and an opportunity to provide feedback on the project and toolkit. This was a great event with some really good feedback on features we should be prioritising and how we're involving developers outside of the project.
+
+Since the start of the year we've also released over 7 new versions, with plenty of new features and bug fixes.
```

Comment:
Maybe add here a few examples of new features (e.g. brjs from anywhere, custom templates).